lib/composite.js

```diff
@@ -2,6 +2,26 @@
 
 const is = require('./is');
 
+/**
+ * Blend modes when compositing images
+ * @member
+ * @private
+ */
+const blendMode = {
```

Comment:
Please can these match libvips' VipsBlendMode [nicknames](https://github.com/libvips/libvips/blob/7da31386c73cacc17116529463529014f8525cc1/libvips/iofuncs/enumtypes.c#L385-L409) (see another of my comments below about how this might be achieved).